lib/gateways/applications-api.ts

```diff
@@ -34,6 +34,24 @@ export const getApplication = async (id: string): Promise<Application | null> =>
   }
 }
 
+export const addApplication = async (application: any): Promise<Application | null> => {
+  try {
+    const headers = {
+      'x-api-key': process.env.AWS_KEY,
+      'Content-Type': 'application/json',
+    }
+    const { data } = await axios.post(
+      `${process.env.ENDPOINT_API}/applications`, application,
+      {
+        headers: headers
+      }
+    );
+    return data;
+  } catch (err) {
+    return null;
```

Comment:
Do we need care for the potential error or is this just a placeholder for now?

Reply:
Error handling in general will need updating, so I think this is ok for now.